.dockerignore

```diff
@@ -0,0 +1,2 @@
+*
+!pkg/docker
```

Comment:
I think is more correct to include the trailing `/` (`!pkg/docker/`) to indicate it's a directory.